src/Agent.Worker/TestResults/ResultsCommandExtension.cs

```diff
@@ -260,36 +257,39 @@ private PublishOptions GetPublishOptions()
             return publishOptions;
         }
 
-        private async Task PublishTestRunDataAsync(VssConnection connection, String teamProject, TestRunContext testRunContext)
```

Comment:
interesting. I take it there's only one usage?

Reply:
yup, only one call